main.cpp

```diff
@@ -63,11 +69,25 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int number2 = 5;
+    int number3 = 8;
+    float f1 = 5.0f;
+    float f2 = 7.89f;
+    float f3 = 0.24f;
+    bool b1 = true;
+    bool b2 = false;
+    bool b3 = 1;
```

Comment:
Do not initialize Boolean variables with integers.  Only use `true` and `false`.
- [ ] revise